cli/server/previewServer.js

```diff
@@ -30,6 +30,26 @@ const PUBLIC_CONFIG_BASENAME = 'cache-public.config';
 const PUBLIC_CONFIG_PATH = path.join(CACHE_DIRECTORY, PUBLIC_CONFIG_BASENAME);
 const FAV_ICON_PATH = './node_modules/jest-preview/cli/server/favicon.ico';
 
+const updateNotifier = require('update-notifier');
+const chalk = require('chalk'); // Already installed in `update-notifier`
+
+// Checks for available update
+const notifier = updateNotifier({
+  pkg: require('../../package.json'),
+  updateCheckInterval: 0, // How often to check for updates
+  shouldNotifyInNpmScript: true, // Allows notification to be shown when running as an npm script
+  distTag: 'latest' // Can be use to notify user about pre-relase version
+})
+
+notifier.notify({
+  defer: true, // Try not to annoy user by showing the notification after the process has exited
+  message:[
+      `${chalk.blue('{packageName}')} has an update available: ${chalk.gray('{currentVersion}')} → ${chalk.green('{latestVersion}')}`,
+      `Please run ${chalk.cyan('`{updateCommand}`')} to update.`,
```

Comment:
@huynhducduy As I see in the PR description, it shows `yarn`. But if I use `npm`, it will show instruction for `npm`, right?
Do you know how it works?

Reply:
@nvh95 Yes it will show the instruction for npm. reference: https://github.com/yeoman/update-notifier/pull/132

Reply:
Thanks for the reference. You had really good skills at investigating stuff.